Sketch.java

```diff
@@ -16,20 +21,63 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(161, 185, 207);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
+
+    float flowerX = 300;
+    float flowerY = 270;
+    float petalSize = 50;
+    float petalDistance = petalSize / 2;
+
+    fill(195, 149, 199);
+
+// upper-left petal
+    ellipse(flowerX - petalDistance, flowerY - petalDistance,
+        petalSize, petalSize);
+
+// upper-right petal
+    ellipse(flowerX + petalDistance, flowerY - petalDistance,
+        petalSize, petalSize);
+
+// lower-left petal
+    ellipse(flowerX - petalDistance, flowerY + petalDistance,
+        petalSize, petalSize);
+
+// lower-right petal
+    ellipse(flowerX + petalDistance, flowerY + petalDistance,
+        petalSize, petalSize);
+
+// center petal
+    fill(248, 252, 131);
+    ellipse(flowerX, flowerY,
+        petalSize, petalSize);
+
+// flower stem
     stroke(128);
-    line(150, 25, 270, 350);  
+    strokeWeight(3/2);
+    line(300, 400, 300, 305);
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+// rectangle of the house
+    fill(189, 154, 102);
+    rect(50, 149, 180, 250);
+    
```

Comment:
align your comments to the code they refer to.  They should not be outdented from the corresponding code.